jdi-bdd-tests/src/test/java/cucumberTests/stepdefs/ScenarioHooks.java

```diff
@@ -0,0 +1,31 @@
+package cucumberTests.stepdefs;
+
+import com.epam.jdi.light.settings.WebSettings;
+import cucumber.api.java.Before;
+import io.github.com.StaticSite;
+
+import static com.epam.jdi.light.elements.init.PageFactory.initElements;
+import static com.epam.jdi.light.settings.WebSettings.logger;
+import static io.github.com.StaticSite.homePage;
+import static io.github.com.entities.Users.DEFAULT_USER;
+import static io.github.com.pages.Header.loginForm;
+import static io.github.com.pages.Header.userIcon;
+import static io.github.com.pages.Header.userName;
+
+public class ScenarioHooks {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The utility class name 'ScenarioHooks' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper)'](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5001515)